### PR TITLE
Skellam

### PR DESCRIPTION
**Pull request by HassanAmr**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-skellam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package:
+  name: r-skellam
+  version: "0.2.0"
+
+source:
+  fn: skellam_0.2.0.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/skellam_0.2.0.tar.gz
```
Comment (bgruening) [inline recipes/r-skellam/meta.yaml]: please add a checksum

Comment (HassanAmr) [inline recipes/r-skellam/meta.yaml]: There is a checksum in the r builder, should I still add one?

Comment (HassanAmr) [inline recipes/r-skellam/meta.yaml]:
```
* installing *source* package ‘skellam’ ...
** package ‘skellam’ successfully unpacked and MD5 sums checked
** R
** inst
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* creating tarball
packaged installation of ‘skellam’ as ‘skellam_0.2.0.tgz’
* DONE (skellam)
```

Comment (bgruening) [inline recipes/r-skellam/meta.yaml]: r builder? Every URL should have a checksum.

Comment (HassanAmr) [inline recipes/r-skellam/meta.yaml]: ok, pushing now

Comment (HassanAmr) [inline recipes/r-skellam/meta.yaml]: r builder is the name for whatever `$R CMD INSTALL --build .` does :D

File: recipes/r-skellam/meta.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ source:
   fn: skellam_0.2.0.tar.gz
   url:
     - https://cran.r-project.org/src/contrib/skellam_0.2.0.tar.gz
+    md5: e345942d5ccbfb718f08a4affe78d93a
```
Comment (bgruening) [inline recipes/r-skellam/meta.yaml]: the indention is wrong

**Pull request merged**